Dockerfile

```diff
@@ -26,7 +28,19 @@ ENV NODE_ENV=production
 WORKDIR /build
 COPY --from=base /base ./
 
-RUN mkdir -p /usr/local/share/ca-certificates/ && echo ${HOSTALIAS_CERT} | sed 's/\\n/\n/g' | xargs > /usr/local/share/ca-certificates/env.crt && chmod 644 /usr/local/share/ca-certificates/env.crt && npm run build
+RUN mkdir -p /usr/local/share/ca-certificates/ && \
+echo ${HOSTALIAS_CERT} | \
+sed 's/\\n/\n/g' | \
+xargs > \
+/usr/local/share/ca-certificates/env.crt && \
+chmod 644 /usr/local/share/ca-certificates/env.crt && \
+mkdir -p  /etc/ssl/certs/ && \
+echo ${HOSTALIAS_ROOT_CERT} | \
+sed 's/\\n/\n/g' | \
+xargs > \
+/etc/ssl/certs/root.crt && \
+chmod 644  /etc/ssl/certs/root.crt && \
+npm run build
 
 FROM node:20-alpine3.20 AS production
```

Comment:
I know this isn't my project, but I accidentally clicked on this PR so thought I'd take a peek.

You should really pin the base image version, for build reproducibility.

```
FROM node:20.16.0-alpine3.19 AS production
```

If you don't, the build you get is not guaranteed to be the same each time.

Reply:
Fair point, thanks for bringing it up. Should be updated now.